### PR TITLE
Akash scrum64

### PR DESCRIPTION
**Pull request by akashmisra1**

### CODE REVIEW

File: src/app/education/page.tsx [hunk 1]
```diff
@@ -231,6 +235,100 @@ export default function Home() {
...
+            >
+              <ModalContent>
+                <>
+                  <ModalHeader className="bg-blue-300">
+                    {" "}
+                    How can Vocoders be Used?{" "}
+                  </ModalHeader>
+                  <ModalBody>
+                    <p>
+                      Now that we have talked about the versitality of vocoders,
```
Comment (andrewpeng02) [inline src/app/education/page.tsx]: small typo "versatility"

**Review by andrewpeng02 — COMMENTED** [review]
Should vocoder_learning_content be updated?

**Comment by akashmisra1** [comment]
Fixing the download to pdf should be its own ticket since this issue persists across all learning modals and not just this one

**Review by buchanmatthew — APPROVED** [review]
great work

**Pull request merged**